pydash-front/README.md

```diff
@@ -0,0 +1,2434 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
reduce this Readme, as this 2,4k lines readme is not important for the users of the project, only for the devs